pkg/test/docker.go

```diff
@@ -180,3 +195,46 @@ func (d *dockerRunner) RemoveAllContainers() {
 		}
 	}
 }
+
+func (d *dockerRunner) setHostMapping(resource *dockertest.Resource, mapping hostMapping) error {
+	timeout := time.Duration(1) * time.Millisecond
```

Comment:
i think this is way too fast. In your mind you may think that we are only handling binary applications so it must be this fast, but what about programs that are not as fast (java, php, ...)?
```suggestion
	timeout := time.Duration(100) * time.Millisecond
```